src/pydata_sphinx_theme/theme/pydata_sphinx_theme/sections/sidebar-secondary.html

```diff
@@ -1,7 +1,7 @@
 {% block docs_toc %}
 {# Define content / TOC behavior depending on whether we have sidebars #}
 {% set toc_show_width = "xl" if sidebars else "md" %}
-<div class="bd-sidebar-secondary d-none d-{{ toc_show_width }}-block col-{{ toc_show_width }}-2 bd-toc">
+<div class="bd-sidebar-secondary col-6 col-lg-2">
```

Comment:
no more `bd-toc`? Isn't that a selector we use in many places? Also is this deprecating the `toc_show_width` functionality?